packages/ui/src/Tooltip.module.scss

```diff
@@ -1,26 +1,134 @@
 @use '../styles/constants' as c;
 
-.wrapper {
+.overlay {
+  @include c.typographyBodySmall;
+
+  position: relative;
   max-width: c.$tooltipMaxWidth;
-  word-wrap: break-word;
-}
+  padding: c.$grid * 5;
+  background: c.$colorNeutralLighter;
+  border: c.$borderWidth solid c.$colorNeutralLight;
+  border-radius: c.$borderRadius;
+  font-size: inherit;
+  color: c.$colorText;
 
-.overlay {
-  // TODO: Update colors. Ask Pawel.
-  color: #fff;
+  &.hidden {
+    clip: rect(0 0 0 0);
+    clip-path: inset(50%);
+    overflow: hidden;
+    position: absolute;
+    white-space: nowrap;
+  }
```

Comment:
nit: Wouldn't simple `visibility: hidden;` suffice? I did some smoke-testing using it and it appears to work as expected.